sources/packages/web/src/views/student/StudentAppealRequest.vue

```diff
@@ -5,51 +5,51 @@
     </template>
 
     <div v-if="showRequestForAppeal">
-      <!-- content area -->
-      <div>
-        <formio
-          formName="studentrequestchange"
-          @loaded="formLoaded"
-          @submitted="submitRequest"
-        ></formio>
-      </div>
-      <!-- action area -->
-      <div class="mt-4">
-        <v-btn @click="submitStudentRequest" color="primary" class="float-right"
-          >Next</v-btn
-        >
-      </div>
+      <!-- Content area. -->
+      <formio-container
+        formName="studentRequestChange"
+        @loaded="formLoaded"
+        @submitted="submitRequest"
+      >
+        <template #actions="{ submit }">
+          <v-btn @click="submit" color="primary" class="float-right">
+            Next
+          </v-btn>
+        </template>
+      </formio-container>
+
+      <!-- Action area. -->
+      <div class="mt-4"></div>
```

Comment:
I am not following why do we need an empty `div` here. Can you please explain?

Reply:
that's a left over. removing